assets/l10n/app_en.arb

```diff
@@ -529,6 +533,10 @@
   "@userRoleUnknown": {
     "description": "Label for UserRole.unknown"
   },
+  "inboxPageTitle": "Inbox",
+  "@inboxPageTitle": {
+    "description": "Title for the page of inbox."
```

Comment:
This description doesn't read very well to me. But then I realized the same is true of some of the existing descriptions of similar strings, so I sent a quick PR fixing those:
* #1109 

This can then follow the style of those.